Patterns/Dutch/Dutch-DateTime.yaml

```diff
@@ -1155,7 +1155,8 @@ HolidayNames: !dictionary
     ascensionday: [ hemelvaartsdag ] # 39 Days after Easter
     whitesunday: [ pinksteren ]
     christmas: [ kerstfeest, kerstmis, kerst, xmas ]
-    easterday: [ pasen, paasdag ]
+    easterday: [ pasen, paasdag, paaszondag ]
+    eastermonday: [ paasmaandag ]
```

Comment:
Would be good to have `tweede paasdag` here as well, since this is more common.

Reply:
ok, added

Reply:
Thanks!